### PR TITLE
[doc]: PCF Documentation update

### PR DESCRIPTION
**Pull request by ajithsimons**
## Description
- PCF Documentation update

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by ajithsimons** [comment]
Please Merge this PR after #43 

@AjayRaturi123  Please review it.

@FaGru3n Please merge after review.

File: Dockerfile [hunk 1]
```diff
@@ -48,4 +48,5 @@ RUN chmod ug+rwx /usr/share/nginx/html/
 EXPOSE 8080
 USER 101
 # Start Nginx server
-CMD ["/bin/sh", "-c", "/usr/share/nginx/html/env.sh && nginx -g \"daemon off;\""]
+CMD ["nginx", "-g", "daemon off;"]
+#CMD ["/bin/bash", "-c", "/usr/share/nginx/html/env.sh && nginx -g \"daemon off;\""]
```
Comment (dvasunin) [inline Dockerfile:52]: better to remove commented line, anyway you can always look for it in git history

**Review by dvasunin — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (ajithsimons) [reply Dockerfile:52]: @adityagajbhiye9  pls check.

Comment (adityagajbhiye9) [reply Dockerfile:52]: I have removed in PR #45 .

Comment (adityagajbhiye9) [reply Dockerfile:52]: @dvasunin  Please review.

Comment (dvasunin) [reply Dockerfile:52]: already approved

**Review by adityagajbhiye9 — APPROVED** [review]
LGTM

**Review by FaGru3n — CHANGES_REQUESTED** [review]
please remove the docker file changes from our pr

**Review by adityagajbhiye9 — APPROVED** [review]
LGTM.

**Pull request merged**